### ISSUE TITLE
Request for contribution

### ISSUE THREAD

**Issue by sniperadmin** [Kinrany/vue-p5#34]
I do love the idea of adding a vue support for p5 framework!

I noticed that there is a ton of work required, I have setup unit testing for the current implementation and some of the missing features and requirements that we need...

I will be glad for joining!!

**Comment by Kinrany** [comment]
Yeah, there's a long list of thing I wanted to do for a long time but never had the time to do. PRs are welcome!

**Comment by sniperadmin** [comment]
Thank you so much!

**Comment by sniperadmin** [comment]
I could suggest one of these apps or actions to check for the CLA

I might have rushed into it, but it might be useful in the future

https://github.com/marketplace?query=cla+
